Mvp2/export import page state (#93)
Knotx/knotx-fragments-chrome-extension: README.md

* [Babel](https://babeljs.io/)
* [Eslint](https://eslint.org/)
* [Webpack](https://webpack.js.org/)

> * [SingleFile](https://github.com/gildas-lormeau/SingleFileyg )

Reviewer: ```suggestion
* [SingleFile](https://github.com/gildas-lormeau/SingleFile )
```
Typo

The pageData entry is created on page load and destroyed when we close the tab. If the page does not contain Knot.x fragments, fragments property is empty.

## Page dump functionality

> We provide page dump functionality to allow easier bug reproducing on another machine. User can downlad current state of page when he find a bug, instead of refreshing the page in hope that the bug disappear.

Reviewer: ```suggestion
We provide a page dump functionality to allow easier bug reproducing on different machines. Users can download the current state of the page when a bug occurs. It's especially useful for nondeterministic bugs that can disappear after refreshing the page.
```

> To provide this functionality we include SingleFile extension in our extension. Information about how to include SingleFile API into extension you can find [here](https://github.com/gildas-lormeau/SingleFile/wiki/How-to-integrate-the-API-of-SingleFile-into-an-extension)

Reviewer: ```suggestion
To provide this functionality we include the SingleFile extension. The documentation on integrating the SingleFile API with other extensions can be found [here](https://github.com/gildas-lormeau/SingleFile/wiki/How-to-integrate-the-API-of-SingleFile-into-an-extension)
```

> User can dump page by click on 'dump page button' in extention popup. This click fire the following data flow:

Reviewer: ```suggestion
Users can dump a page by clicking on the 'dump page button' in the extension popup. It fires the following data flow:
```

```
popup -> background -> content -> popup

```

> dump function you can find in ```/src/js/content/dump.js```

Reviewer: ```suggestion
Dump implementation can be found in ```/src/js/content/dump.js```
```
